app/templates/courses/teacher-class-view.jade

```diff
@@ -388,7 +203,7 @@ mixin studentLevelsRow(student)
             - var progress = state.get('progressData').get({ classroom: view.classroom, course: course, level: level, user: student })
             - var levelNumber = view.classroom.getLevelNumber(level.get('original'), index + 1)
             - if (!level.get('practice'))
-              - courseFinished = progress.completed;
+            - courseFinished = progress.completed;
```

Comment:
It looks like a typo to remove the indentation here, should be inside `if` on the above line I believe.

Reply:
Yep - updated